roles/kubernetes-apps/ansible/templates/dashboard.yml.j2

```diff
@@ -174,7 +191,7 @@ spec:
       volumes:
       - name: kubernetes-dashboard-certs
         secret:
-          secretName: {{ dashboard_certs_secret_name }}
```

Comment:
Can you revert this change? The idea is to keep the cert secret configurable

Reply:
ok.  I add a judge in here.